src/components/NavBar/index.jsx

```diff
@@ -52,6 +52,9 @@ const NavBar = ({ isTesting }) => {
               <Icon style={{ transform: "scale(2)" }} component={AdminSvg} />
             </Link>
           </Menu.Item>
+          <Menu.Item key="6" className="navbarLogin">
+            <Link to="/register">Register</Link>
```

Comment:
Can you change this to sign-in/sign-up, or maybe use a conditional to show either of them? And make sure to add a sign out button somewhere too after the user logs in.

Reply:
I changed the name as Sign-In / Sign-Up for now. Maybe we can change the icon and we can move the sign in sign out section on the right side of the navbar.